src/main/java/presentation/Ui.java

```diff
@@ -0,0 +1,87 @@
+package presentation;
+
+import tasks.Task;
+
+import java.util.ArrayList;
+
+/** A class that display to the user */
+public class Ui {
+
+    /** Creates an instance of Ui object. */
+    public Ui() {
+    }
+
+    /**
+     * Shows greeting dialog when the application starts.
+     */
+    public void greetDialog() {
+        String logo =
+                " __\n"
+                        + "| |    _   _ _  _____\n"
+                        + "| |   | | | | |/ / _ \\\n"
+                        + "| |__ | |_| |   <  __/\n"
+                        + "|____| \\__,_|_|\\_\\___|\n";
+        System.out.println("Hello from\n" + logo);
+        System.out.println("Yo! I'm Luke.\nHow's it hanging?");
+    }
+
+    /**
+     * Shows closing dialog when the application ends.
+     */
+    public void closingDialog() {
```

Comment:
Would it be clearer to have your method be named "closeDialog" without the adverb?